dashboard/app/models/pd/application/teacher_application.rb

```diff
@@ -1147,10 +1148,10 @@ def total_score
     end
 
     # Called after the application is created. Do any manipulation needed for the form data
-    # hash here, as well as wend emails
+    # hash here, as well as send emails
     def on_successful_create
       update_user_school_info!
```

Comment:
we might not want to call this method either, yet, as the school data could potentially be incomplete.

Reply:
maybe only call on_successful_create at all only on a complete submitted application

Reply:
Yep, good call

Reply:
Just left a comment for now